### PR TITLE
[Keyboard] [runner3680] Fix the number of keys

### PR DESCRIPTION
**Pull request by omkbd**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Fix the number of keys
 1. The number of matrix rows and cols
 2. The number of RGBLED
* Add RGBLIGHT_LIMIT_VAL

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/runner3680/4x7/config.h [hunk 1]
```diff
@@ -51,9 +51,10 @@
 /* ws2812 RGB LED */
 #define RGB_DI_PIN D3
 #define RGBLIGHT_ANIMATIONS
-#define RGBLED_NUM 70
+#define RGBLED_NUM 28
```
Comment (noroadsleft) [inline keyboards/runner3680/4x7/config.h]: Is this accurate? Most other variants have this set to `MATRIX_ROWS * MATRIX_COLS`.

Seems like this file should have

```
#define RGBLED_NUM 56
...
#define RGBLED_SPLIT { 28, 28 }
```

File: keyboards/runner3680/5x6/config.h [hunk 2]
```diff
@@ -27,7 +27,7 @@
 /* key matrix size */
 // Rows are doubled-up
 #define MATRIX_ROWS 10
-#define MATRIX_COLS 8
+#define MATRIX_COLS 12
```
Comment (noroadsleft) [inline keyboards/runner3680/5x6/config.h]: I believe this should be:

```suggestion
#define MATRIX_COLS 6
```

File: keyboards/runner3680/4x7/config.h (hunk 1 above)
Comment (omkbd) [reply keyboards/runner3680/4x7/config.h]: Thank you. You are right.

File: keyboards/runner3680/5x6/config.h (hunk 2 above)
Comment (omkbd) [reply keyboards/runner3680/5x6/config.h]: Thank you.

**Pull request merged**